File: postman/sdks/Specs SDKs/Orders/cli/sdk/orders/order_list_response.go
package orders

import (
	"encoding/json"
	"example.com/orders/sdk/internal/unmarshal"
)

type OrderListResponse struct {
	// Orders in this page
	Data []Order `json:"data" xml:"data" required:"true"`
	// Number of orders returned
	Count int64 `json:"count" xml:"count" required:"true" min:"0"`
}

func (o OrderListResponse) String() string {
	jsonData, err := json.MarshalIndent(o, "", "  ")
	if err != nil {
		return "error converting struct: OrderListResponse to string"
	}
	return string(jsonData)
}

func (o *OrderListResponse) UnmarshalJSON(data []byte) error {
	if err := unmarshal.ValidateRequiredJSONKeys(data, o); err != nil {
		return err
	}
	type alias OrderListResponse
	var tmp alias
	if err := json.Unmarshal(data, &tmp); err != nil {
		return err
	}
	*o = OrderListResponse(tmp)
	return nil
}

File: postman/sdks/Specs SDKs/Orders/python/src/orders_sdk/models/order_list_response.py
from __future__ import annotations
from typing import List
from pydantic import Field
from typing import Optional
from typing import Any
from .utils.base_model import BaseModel
from .order import Order


class OrderListResponse(BaseModel):
    """OrderListResponse

    :param data: Orders in this page
    :type data: List[Order]
    :param count: Number of orders returned
    :type count: int
    """

    data: List[Order] = Field(description="Orders in this page")
    count: int = Field(description="Number of orders returned", ge=0)
